pyproject.toml

```diff
@@ -2,9 +2,6 @@
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-packages = ["BootstrapReport"]
-
 [project]
 name = "BootstrapReport"
 version = "0.0.1" 
```

Comment:
I think we're now at [version 1.0.0](https://github.com/JMSLab/BootstrapReport/releases/tag/v1.0.0)?

Reply:
Addressed in 23e239b !